src/OpenLoco/Types.hpp

```diff
@@ -151,9 +151,17 @@ namespace OpenLoco
             return (static_cast<uint32_t>(wholeNumber) << 16) | static_cast<uint16_t>((fraction << 16) / 100000);
         }
 
+        static_assert(0.0_mph == 0x0000);
+        static_assert(1.0_mph == 0x10000);
         static_assert(2.75_mph == 0x2C000);
         static_assert(4.0_mph == 0x40000);
+        static_assert(6.0_mph == 0x60000);
+        static_assert(9.0_mph == 0x90000);
+        static_assert(13.0_mph == 0xD0000);
```

Comment:
Not all of these need to be included as a test case, imo; it's all variants of the same. @duncanspumpkin, what are your thoughts?

Reply:
I don't mind a few extra. Now that you have confirmed these are identical though they can be deleted.

Reply:
Ok, I removed them.